mpas_analysis/ocean/meridional_heat_transport.py

```diff
@@ -217,13 +220,15 @@ def meridional_heat_transport(config):  # {{{
         #(colormapName, colorbarLevels) = setup_colormap(config, sectionName,
         #                                                suffix=region)
     
-        ##x = read this: ncdump -v binBoundaryMerHeatTrans  mpaso.hist.am.meridionalHeatTransport.0050-12-01.nc
-        ##y = read z-level depth from somewhere.
-        #x = arange(shape(meridionalHeatTransportLatZ,0))
-        #y = arange(shape(meridionalHeatTransportLatZ,1)) 
-        #z = meridionalHeatTransportLatZ 
-        #plot_vertical_section(config, x, y, z, colormapName, colorbarLevels,
-        #                      contourLevels, colorbarLabel, title,
-        #                      xLabel, yLabel, figureName)
+        contourLevels = np.arange(-0.3,0.31,0.1)
+        colorbarLevels = np.arange(-0.3,0.31,0.05)
+        colormapName = plt.get_cmap('seismic')
```

Comment:
Typically, we make these option that users can choose in the config file, rather than hard coding them.  I would suggest doing that in this PR, but we could also do a follow-up PR to change this later if you prefer.  @milenaveneziani, maybe you can help with this?